KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,47 @@
+# Kubeflow Outreach Committee (KOC)
+
+The Kubeflow Outreach Committee (KOC) is a committee dedicated to fostering growth, engagement, and community outreach for the Kubeflow project. It will focus on activities that promote Kubeflow to new and existing users, contributors, and stakeholders while building an inclusive, vibrant, and diverse community within the broader AI/ML ecosystem.
+
+The governance of the Kubeflow Outreach Committee will evolve as the community grows, ensuring that outreach activities remain relevant and impactful.
+
+### Charter
+
+1. Promote the vision, values, and mission of the Kubeflow project.
+2. Lead efforts to grow and engage the AI/ML community, including organizing events, producing educational materials, and advocating for Kubeflow across various tech communities.
+3. Help onboard new contributors and users to Kubeflow, particularly through accessible resources and community-building initiatives.
+4. Collaborate with other committees and working groups to align outreach efforts with the broader goals of the Kubeflow project.
+
+### Committee Structure
+
+- **Appointment of Members**: Members of the Kubeflow Outreach Committee are appointed at the discretion of the Kubeflow Steering Committee (KSC), with the removal of any members for violating the CNCF Code of Conduct also at the KSC's discretion.
+- **Role of KOC Co-Leads**: The Outreach Committee will be co-led by established Kubeflow community members, with the co-leads being responsible for guiding the committee’s work. New members of the KOC may be newcomers to the community, and their primary role will be focused on helping drive growth for Kubeflow.
```

Comment:
We should clarify the difference between a "lead" and "member" of the KOC.

Especially, what are their specific responsibilities/powers of each role.

Reply:
👍